Source/Charts/Animation/ChartAnimationEasing.swift

```diff
@@ -338,11 +338,10 @@ internal struct EasingFunctions
     }
     
     internal static let EaseOutBack = { (elapsed: TimeInterval, duration: TimeInterval) -> Double in
-        let s: TimeInterval = 1.70158
-        var position: TimeInterval = elapsed / duration
-        position -= 1.0
-        let timeInterval: Double = (s + 1.0) * position + s
-        return Double( position * position * timeInterval + 1.0 )
+          let s: TimeInterval = 1.70158
+          var position: TimeInterval = elapsed / duration
+          position -= 1.0
+          return Double( position * position * ((s + Double(1.0)) * position + s) + Double(1.0) )
```

Comment:
Small thing but looks like the spacing is out in a few places
```suggestion
        let s: TimeInterval = 1.70158
        var position: TimeInterval = elapsed / duration
        position -= 1.0
        return Double(position * position * ((s + Double(1.0)) * position + s) + Double(1.0))
```